packages/story-editor/src/app/highlights/states.js

```diff
@@ -102,6 +103,10 @@ export const STATES = {
     focus: true,
     tab: DESIGN,
   },
+  [keys.SIZE_AND_POSITION]: {
```

Comment:
heads up this is changing in Marcin's sidebar update which isn't merged into main yet. https://github.com/GoogleForCreators/web-stories-wp/pull/10760/files#diff-ed4b4a7647ac8a4bb4a9d16aaf241422d72b46cf0fc6af9bd2211cbb7bdc1c55L103

Reply:
I'll update that when it gets merged in 👍